src/hooks/posters.tsx

```diff
@@ -1,5 +1,11 @@
-import { useState, useMemo, useCallback } from "react";
-import { VenueTemplate } from "types/venues";
+import React, {
+  useState,
+  useMemo,
+  useCallback,
+  useContext,
+  createContext,
+} from "react";
+import { VenueTemplate, PosterPageVenue } from "types/venues";
```

Comment:
`types/*` imports after `settings`, before `utils/*`

Reply:
Any reasoning, comment, code style docs that can be referred to as to this kind of rules not seem arbitrary?

Reply:
done.

although I guess I should collect links to comments like these for code style guide, or at least examples for do's and don'ts

Reply:
There is a backlog issue about enabling linting for it that also has the full list in order as I described it once.

As far as collecting links to comments like this. There’s a super old PR here that mentioned documentation that’s comments have sort of ended up as a little bit of a de facto dumping ground for references to comments like this about code style/etc. With the hope/intent of getting the tone to turn that into proper doco.

It would be great if you could keep notes/references as well (ideally somewhere accessible, not just locked away on your computer/similar), would make the process way easier when it comes time to make those docs.